apps/www/_blog/2022-12-16-launch-week-6-community-day.mdx

```diff
@@ -0,0 +1,191 @@
+
+### FlutterFlow
+
+FlutterFlow is a low-code builder for developing native mobile applications using Flutter. You can use the simple drag-and-drop interface to build your app faster than traditional development.
+
+An integration between FlutterFlow and Supabase has been long awaited by the community. Today, you can start building your apps on FlutterFlow having Supabase as the backend!
+
+This is just the beginning of our integration with FlutterFlow, and we will work closely with the FlutterFlow team to bring more features to this integration in the future!
+
+Here is a tutorial video from the FlutterFlow team showing how to get started. 
+
+<iframe
+  className="w-full"
+  width="560"
+  height="315"
+  src="https://www.youtube-nocookie.com/embed/TmViZsWEBY0?modestbranding=1&autohide=1&showinfo=0&controls=0"
+  title="YouTube video player"
+  frameborder="0"
+  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+  allowfullscreen
+></iframe>
+
+
+### OneSignal
+
+OneSignal is a messaging platform allowing you to deliver push notifications, in-app messages, SMS, and emails to your users. Combine OneSignal with Supabase and you have a
+
+Now with combining OneSignal with Supabase, you can send cloud messages to your users whenever there is a change in your Supabase database. 
+
+We have guides on how to get started with Next.js and OneSignal here, and Flutter and OneSignal [here](https://github.com/OneSignalDevelopers/onesignal-supabase-sample-integration-supabase).
+
+William from OneSignal is here to discuss how you can easily integrate OneAignal into your Supabase app.
+
+<iframe
+  className="w-full"
+  width="560"
+  height="315"
+  src="https://www.youtube-nocookie.com/embed/In1eU6J17A8?modestbranding=1&autohide=1&showinfo=0&controls=0"
+  title="YouTube video player"
+  frameborder="0"
+  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+  allowfullscreen
+></iframe>
+
+### NextAuth
+
+[NextAuth.js](https://next-auth.js.org/) is an open source authentication solution for [Next.js](https://nextjs.org/). It allows you to bring your own Database and therefore own your own user data.
+
+There is an awesome open source community around it who have built a [Supabase Adapter](https://next-auth.js.org/adapters/supabase) that makes it easy to use NextAuth.js with Supabase, and even enables usage of [Row Level Security (RLS) policies](https://next-auth.js.org/adapters/supabase#enabling-row-level-security-rls).
+
+In [this community spotlight video](https://youtu.be/EdYQ9fF-hz4), [Thang Huu Vu](https://twitter.com/thanghvu), the lead maintainer of NextAuth.js, gives us an overview and brief history of NextAuth, followed by a tutorial on how to set up the new Supabase Adapter. 
+
+
+<iframe
+  className="w-full"
+  width="560"
+  height="315"
+  src="https://www.youtube-nocookie.com/embed/EdYQ9fF-hz4?modestbranding=1&autohide=1&showinfo=0&controls=0"
```

Comment:
Make sure the video is publically available at the time of the blog post launch.